Formula/p/postgresql@15.rb

```diff
@@ -126,6 +126,13 @@ def install
   end
 
   def post_install
+    unless (HOMEBREW_PREFIX/"bin/psql").exist?
```

Comment:
I may need to check if linked so that a `brew unlink postgresql@15` formula doesn't try to create new links on upgrade

Would this be `linked?` or `prefix_linked?`
```suggestion
    if linked? && !(HOMEBREW_PREFIX/"bin/psql").exist?
```